hw_5.py

```diff
@@ -0,0 +1,11 @@
+﻿"""
```

Comment:
Module name should be also self-descriptive.
`hw_5` tells about nothing.

Reply:
done